### PR TITLE
regions support

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/9839
https://github.com/rancher/rancher/issues/10037
https://github.com/rancher/rancher/issues/10039

### CODE REVIEW

File: infra-templates/ipsec/16/README.md [hunk 1]
```diff
@@ -6,11 +6,10 @@ Rancher networking plugin using IPsec.
...
 Traffic to and from hosts require UDP ports `500` and `4500` to be open.
 
-### Changelog - 0.2.2
+### Changelog - 0.2.3
 
-#### Router and CNI Driver [rancher/net:v0.13.7]
-* Introduced connectivity check for ipsec.
-* Fixed an issue where scheduler IPs could be picked up as the IP for the ipsec service for ports `500` and `4500`, which would cause cross host communication to no longer work for that host
+#### Router, CNI Driver, Connectivity Check [rancher/net:v0.13.8]
+* Added support for regions
```
Comment (deniseschannon) [inline infra-templates/ipsec/16/README.md]: can you add the change log for the logs update? https://github.com/rancher/rancher/issues/7571

File: infra-templates/ipsec/16/rancher-compose.yml [hunk 2]
```diff
@@ -21,6 +21,18 @@
       type: "string"
       default: '10.42.0.0/16'
       required: true
+    - variable: "SUBNET_START_ADDRESS"
+      label: "Subnet start IP Address"
+      description: "Proceed with caution; typical users will not need to change these. Incorrect values can break your Rancher installation."
```
Comment (deniseschannon) [inline infra-templates/ipsec/16/rancher-compose.yml]: Let's reword to: `This is an advanced setting and most users will not need to use this. Incorrect values can break your Rancher installation.`

File: infra-templates/ipsec/16/rancher-compose.yml [hunk 3]
```diff
@@ -21,6 +21,18 @@
...
+      type: "string"
+      default: ''
+      required: false
+    - variable: "SUBNET_END_ADDRESS"
+      label: "Subnet end IP Address"
+      description: "Proceed with caution; typical users will not need to change these. Incorrect values can break your Rancher installation."
```
Comment (deniseschannon) [inline infra-templates/ipsec/16/rancher-compose.yml]: Let's reword to: `This is an advanced setting and most users will not need to use this. Incorrect values can break your Rancher installation.`

File: infra-templates/ipsec/16/rancher-compose.yml [hunk 4]
```diff
@@ -69,7 +81,12 @@
       type: "string"
       default: '1h'
       required: true
-
+    - variable: "RANCHER_IPSEC_PSK"
+      label: "PSK for IPSec"
+      description: "Proceed with caution; typical users will not need to change this. Incorrect values can break your Rancher installation."
```
Comment (deniseschannon) [inline infra-templates/ipsec/16/rancher-compose.yml]: Let's reword to: `This is an advanced setting and most users will not need to use this. Incorrect values can break your Rancher installation.`

File: infra-templates/ipsec/config.yml.final [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name: Rancher IPsec
```
Comment (deniseschannon) [inline infra-templates/ipsec/config.yml.final]: Why is this file name `config.yml.final`? can you update the `config.yml`?

File: infra-templates/network-policy-manager/2/rancher-compose.yml [hunk 6]
```diff
@@ -1,4 +1,4 @@
 .catalog:
   name: Network Policy Manager
-  version: v0.0.2
-  minimum_rancher_version: v1.5.0-rc1
+  version: v0.0.3
+  minimum_rancher_version: v1.6.15-rc1
```
Comment (deniseschannon) [inline infra-templates/network-policy-manager/2/rancher-compose.yml]: Can you add a README for the changes for this infrastructure service?

File: infra-templates/network-services/27/README.md [hunk 7]
```diff
@@ -8,10 +8,11 @@ This stack provides the following services:
 
 ### Changelog for v0.2.8
```
Comment (deniseschannon) [inline infra-templates/network-services/27/README.md]: Update to `v0.2.9`

**Review by deniseschannon — CHANGES_REQUESTED** [review]

**Comment by leodotcloud** [comment]
@deniseschannon took care of the review comments, can you please check again?

**Review by deniseschannon — APPROVED** [review]

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**